<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>540</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QListView" name="border_image">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>700</width>
     <height>370</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>370</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>370</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/school.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;
color: rgb(0, 0, 235);</string>
   </property>
   <property name="text">
    <string>基于三维模型的车辆位姿估计标注系统</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListView" name="calibration">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/calibration.png);</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>761</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>287</width>
     <height>231</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>287</width>
     <height>231</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>287</width>
     <height>231</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255,0.5);</string>
   </property>
   <layout class="QFormLayout" name="enter">
    <property name="rowWrapPolicy">
     <enum>QFormLayout::WrapLongRows</enum>
    </property>
    <property name="horizontalSpacing">
     <number>20</number>
    </property>
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="number">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>UpArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255,255,255,o);</string>
      </property>
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="number_in">
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>161</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="password">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>UpArrowCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255,255,255,0);</string>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="password_in">
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>161</width>
        <height>21</height>
       </size>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="phone">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>UpArrowCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255,255,255,0);</string>
      </property>
      <property name="text">
       <string>电话</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="phone_in">
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>161</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="registerbutton">
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(244, 255, 255);</string>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="cancelregisterbutton">
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(244, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
